imgui_impl_vitagl.cpp

```diff
@@ -382,25 +392,35 @@ void ImGui_ImplVitaGL_NewFrame()
 
 	// Touch for mouse emulation
 	if (touch_usage){
-		SceTouchData touch;
-		sceTouchPeek(SCE_TOUCH_PORT_FRONT, &touch, 1);
-		if (touch.reportNum > 0){
-			mx = lerp(touch.report[0].x, 1920, 960);
-			my = lerp(touch.report[0].y, 1088, 544);
-		}
+		double scale_x = 960.0 / io.DisplaySize.x;
+		double scale_y = 544.0 / io.DisplaySize.y;
+		double offset_x = 0;
+		double offset_y = 0;
+		ImGui_ImplVitaGL_PollTouch(offset_x, offset_y, scale_x, scale_y, &mx, &my, g_MousePressed);
 	}
```

Comment:
this ensures the scaling of touch input is correct even when the frame buffer size is changed.